core/src/main/java/org/kohsuke/stapler/IndexHtmlDispatcher.java

```diff
@@ -36,7 +36,7 @@ public String toString() {
      * Returns a {@link IndexHtmlDispatcher} if and only if the said class has {@code index.html} as a side-file
      */
     static Dispatcher make(ServletContext context, Class c) {
```

Comment:
I would suggest documenting the null case in Javadoc/Annotations here and in Calling methods just to make sure nobody misuses the API.